Milestone 1: Index page and Search results page

sinansevgi opened the pull request:
**Description of the change**

- I created an online store page for the bookshop.
- Added index.html and result.html file to project.
- Added styling files and assets to the project.
- I used bootstrap to give responsiveness to the website.

**index.html**

- Index page of the project.
- Lists most selling categories and most selling books of those categories.

**result.html**

- Lists search results as media items.
- Main left section has form-input items that can be used for filtering actions.

sinansevgi committed:
Index Page Created.

-Book covers added.
-Carousel structure created.

sinansevgi committed:
Index Page Completed.

-Footer Menu added.
-Footer Completed.

sinansevgi committed:
Fonts Changed.

-Fonts changed to fit design.
-Background of Jumbotron changed.
-Fonts applied to site design.

sinansevgi committed:
Results page created.

-Results added.

sinansevgi committed:
Linter errors fixed.

sinansevgi committed:
Accesibility errors fixed.

sinansevgi committed:
Read Me file created.

sinansevgi committed:
Price tags for results page added.

-Screenshot added.
-Readme file fixed.

sinansevgi committed:
Accessibility errors on the readme file fixed.

sinansevgi committed:
Github linters added.

sinansevgi committed:
Errors on fonts.css fixed.

bertil291utn commented:
General requirements
() Edit pull request message following this [guideline](https://cdn.filestackcontent.com/osvFJEkQHWQsgDe0PJoQ)
() 
FILE: README.md
()
FILE: index.html
Large screen
()
Medium screen
()
Small screen
()
FILE: search.html
Large screen
()
Medium screen
()
Small screen
()
FILE: result.html
Large screen
()
Medium screen
()
Small screen
()

TO HIGHLIGHT
()

NICE TO HAVE 
()

sinansevgi committed:
License link fixed.

sinansevgi committed:
Fix Design


-Icons added to category headings.
-Item cards redesigned white space removed.
-Pagination links added to result.html.
-Link to index.html added to logo.
-Lorem text replaced.
-result.html pages result items heading changed to h2.
-nav tags colors changed according to original design.
